taxi/views.py

```diff
@@ -33,8 +40,24 @@ def index(request):
 class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     model = Manufacturer
     context_object_name = "manufacturer_list"
-    template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
+    queryset = Manufacturer.objects.all()
```

Comment:
Do you need to define queryset in the class when you override the get_queryset method?